### PR TITLE
fix: upon modifying the information in the Inbox, the list failed to update promptly

### PR DESCRIPTION
**Pull request by cuikaipeng**
### Description

on the Discover page, when modifying information in the Inbox, the list does not update immediately after the modification is completed.

https://github.com/user-attachments/assets/34d975b8-4cd7-4818-91b2-4d733c495c3f

### CODE REVIEW

File: apps/renderer/src/modules/discover/inbox-list-form.tsx [hunk 1]
```diff
@@ -102,7 +102,14 @@ export function DiscoverInboxList() {
                     present({
                       title: t("sidebar.feed_actions.edit_inbox"),
                       content: ({ dismiss }) => (
-                        <InboxForm asWidget id={inbox.id} onSuccess={dismiss} />
+                        <InboxForm
+                          asWidget
+                          id={inbox.id}
+                          onSuccess={() => {
+                            inboxes.refetch()
```
Comment (Innei) [inline apps/renderer/src/modules/discover/inbox-list-form.tsx:109]: Use optimistic update?

Comment (cuikaipeng) [reply apps/renderer/src/modules/discover/inbox-list-form.tsx:109]: ok, I'll optimize it

**Comment by Innei** [comment]
This implementation does not make sense. I will refactor later

**Comment by cuikaipeng** [comment]
> This implementation does not make sense. I will refactor later

sorry to bother you, I would like to ask you for some advice. in useMutation, I handle optimistic updates through onMutate, is this “make sense“ ?   
code like ⬇️
iShot_2024-10-06_17 51 19

**Pull request closed without merging**